.eslintrc

```diff
@@ -8,6 +8,9 @@
 
     "globals": {
         "jsdom": true,
+        "shallow": true,
+        "render": true,
+        "mount": true
```

Comment:
From my research, it must be done like that, but if you find something it can always be changed later, not that it makes much of a difference, as this only affects the dev stage and doesn't impact the application